ee/fe/modules/340-monitoring-applications/applications/rabbitmq/prometheus-rules/rabbitmq.yaml

```diff
@@ -1,7 +1,7 @@
 - name: applications.rabbitmq
   rules:
   - alert: ApplicationRabbitMQMemoryUsage
-    expr: rabbitmq_node_mem_used{job="rabbitmq"} / rabbitmq_node_mem_limit{job="rabbitmq"} * 100 > 85
+    expr: rabbitmq_process_resident_memory_bytes{job="rabbitmq"} / rabbitmq_resident_memory_limit_bytes{job="rabbitmq"} * 100 > 85
```

Comment:
Your changes will break alerts and dashboards if someone uses older versions of RMQ metrics. I think it's better to use PromQL like this:
```
((rabbitmq_node_mem_used{job="rabbitmq"} / rabbitmq_node_mem_limit{job="rabbitmq"}) or (rabbitmq_process_resident_memory_bytes{job="rabbitmq"} / rabbitmq_resident_memory_limit_bytes{job="rabbitmq"})) * 100 > 85
```
This way, we don't break backward compatibility with older versions RMQ metrics.  This also applies to all subsequent changes.